CHANGELOG.md

```diff
@@ -11,6 +11,25 @@
     (that's why ``config`` must expose your webpack configurations)
   - You webpack configuration can now skip some tiny weird part specific to
     statinamic.
+
+- Changed: Remove redux devtools and `process.env.CLIENT` environment variables.
+  To benefits from this, you can:
```

Comment:
"To pull this change" (it's not a benefit since we remove something).

Maybe you can also add this explanation: "Redux will probably become part of the private API, which will reduce the boilerplate. In order to do that, we will gradually remove Redux from all public interface".